src/molecule/verifier/testinfra.py

```diff
@@ -161,7 +161,6 @@ def bake(self):
             cmd=["pytest", *util.dict2args(options), *self._tests, *args],
             cwd=self._config.scenario.directory,
             env=self.env,
-            stdout=LOG.out,
```

Comment:
Why did we leave the `stderr` param in this case while we removed it from other bake methods?

Reply:
I removed stderr also, just missed it.

Background: All the bakes are vestigial from when we used sh library and remained just to lower the size of the change that removed sh. Now that we use subprocess_tee.run to execute subprocesses we no longer need to do tricks with stdout/stderr.

I am not yet in a hurry to cleanup these bakes yet. I want to spread the refactoring across several major releases, so we lower the risks of making really bad release.